spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1112,39 +1016,27 @@ def main(args=None):
         name_str = ' for ' + spot_ros.name
     node.get_logger().info("Starting ROS driver for Spot" + name_str)
     ############## testing with Robot
-    if spot_ros.name != MOCK_HOSTNAME:
+    if spot_ros.name == MOCK_HOSTNAME:
+        spot_ros.spot_wrapper = None
+    else:
         spot_ros.spot_wrapper = SpotWrapper(spot_ros.username, spot_ros.password, spot_ros.ip, spot_ros.name,
                                             spot_ros.logger, spot_ros.start_estop.value, spot_ros.estop_timeout.value,
                                             spot_ros.rates, spot_ros.callbacks, spot_ros.use_take_lease,
                                             spot_ros.get_lease_on_action, spot_ros.continually_try_stand)
         if not spot_ros.spot_wrapper.is_valid:
             return
-    else:
-        spot_ros.spot_wrapper = None
-    # spot_ros.spot_wrapper = spot_wrapper
-    if spot_ros.spot_wrapper is None or spot_ros.spot_wrapper.is_valid:
-        has_arm = False
-        if spot_ros.spot_wrapper is not None:
-            has_arm = spot_ros.spot_wrapper.has_arm()
+
+        all_cameras = ["frontleft", "frontright", "left", "right", "back"]
```

Comment:
This should probably be a list at the top of the file as a constant, or an enum.

Reply:
I think that we should actually ask spot what camera sources it supports, rather than hard-coding it.

Reply:
I agree that would be best. Would have to be done in spot_wrapper

Reply:
Yeah, ideally spot wrapper would provide that.